doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be simple enough for non tech savvy users to use.
+6. Should be able to save data reliably and accurately.
```

Comment:
This one is quite blur! There is already a feature that saves data to the file on each command.